src/messagingsession/MessagingSessionConfiguration.ts

```diff
@@ -30,13 +30,17 @@ export default class MessagingSessionConfiguration {
   /**
    * Constructs a MessagingSessionConfiguration optionally with userArn, messaging session id, a messaging session
    * endpoint URL, and the chimeClient.
+   *
+   * endpointUrl is deprecated as internally it is set on connect via chimeClient if null, and re-resolved (whether null or not) from
```

Comment:
Since you are using `undefined` over `null` could you update this docstring accordingly?

Reply:
updated thx